x-pack/plugins/lists/common/schemas/elastic_query/update_es_list_schema.ts

```diff
@@ -16,6 +16,7 @@ import {
   updated_by,
 } from '../common/schemas';
 
+// TODO: Should we use partial here and everywhere these are instead of this OrUndefined?
```

Comment:
NOTE: Ignore the TODO's as they will be removed or accomplished in future pull requests.

Reply:
Is this somewhere where we could use `RequiredKeepUndefined`?

Reply:
It is local to this project at the moment but it's simple enough to copy it to another project if you want to use it else where. If I see usages in SIEM or a need in SIEM, I would probably copy it there and then import it from SIEM back into this project.

Reply:
Oh wait I might have misunderstood you. Sorry. If this gets changed over to `t.partial` with io-ts partials than yep that would be the goal!

I still want to run more things through decode along API boundaries and do a throw or something to indicate to programmers very early that they have mistakes with their types such as either a saved object type or an elastic type.

Reply:
👍 Yea, that's what I was getting at.

Reply:
Great, perfect, the TODO will be flushed in a follow up PR if this is acceptable as is now, otherwise I will attach it here if I have to make any other changes.